charts/apisix/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- $kubernetesTokenFile := "/var/run/secrets/kubernetes.io/serviceaccount/token" -}}
```

Comment:
So that we can avoid writing them duplicately.

Reply:
No problem, but tpl can only return a string, we would need to work on string instead of currently boolean of `$create`. Another way is to combine the cluster role and binding in one file. Not sure which is better.

Reply:
@tao12345666333 What's your idea, I think it's OK.

Reply:
Create a new file named "rbac.yaml".

Reply:
Forgive my ignorant, I just stumbled across https://helm.sh/docs/chart_best_practices/rbac/ and noticed this pattern:
```
rbac:
  # Specifies whether RBAC resources should be created
  create: true

serviceAccount:
  # Specifies whether a ServiceAccount should be created
  create: true
  # The name of the ServiceAccount to use.
  # If not set and create is true, a name is generated using the fullname template
  name:
```
Maybe we should follow this?

Reply:
I think it's OK.

Reply:
Updated to following https://helm.sh/docs/chart_best_practices/rbac/.